inference-engine/src/inference_engine/ie_core.cpp

```diff
@@ -290,7 +290,9 @@ class Core::Impl : public ICore {
             networkIsImported = false;
         } catch (...) {
             cacheManager->removeCacheEntry(blobId);
-            throw;
+            networkIsImported = false;
+            // TODO: temporary disabled by #54335. In future don't throw only for new 'blob_outdated' exception
+            // throw;
```

Comment:
will not we do the same for `Export`?
Or we expect that Export can fail only in case of issues with FS or with bugs in plugins itself and these cases are OK to throw?

Reply:
I assume that primary use case is to not throw exception when device driver is updated. For this case there is completely no need to catch exception for 'Export'. In case of some 'export-unsupported' operations I still think that caching shall have the same behavior as regular 'export'. But primarily I'm ok if you request to 'catch' export exceptions as well